webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/BinaryDataService.scala

```diff
@@ -20,6 +22,7 @@ class BinaryDataService(val dataBaseDir: Path,
                         val agglomerateServiceOpt: Option[AgglomerateService],
                         dataVaultServiceOpt: Option[DataVaultService],
                         val applicationHealthService: Option[ApplicationHealthService],
+                        sharedChunkContentsCache: Option[AlfuCache[String, MultiArray]],
                         datasetErrorLoggingService: Option[DatasetErrorLoggingService])
     extends FoxImplicits
     with DataSetDeleter
```

Comment:
Doesn't have to be this PR, but I noticed that you renamed DataSet stuff to Dataset. Should be done everywhere, e.g. here.

Reply:
Yes, that’s follow-up for https://github.com/scalableminds/webknossos/issues/6053 – trouble is, we can’t just replace everything, because some parts are serialized and/or used elsewhere (e.g. routes). But for local variables, I intend to use dataset in the future as discussed in the linked issue